src/applications/vaos/components/review/PreferredProviderSection.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Link } from 'react-router';
+import newAppointmentFlow from '../../newAppointmentFlow';
+import { LANGUAGES } from '../../utils/constants';
+
+export default function PreferredProviderSection(props) {
+  return (
+    <>
+      {props.data.hasCommunityCareProvider && (
+        <>
+          <div className="vads-l-grid-container vads-u-padding--0">
+            <div className="vads-l-row">
+              <div className="vads-l-col--6">
+                <h3 className="vaos-appts__block-label">Preferred providers</h3>
+              </div>
+              <div className="vads-l-col--6 vads-u-text-align--right">
+                <Link
+                  to={newAppointmentFlow.ccPreferences.url}
+                  aria-label="Edit provider preference"
+                >
+                  Edit
+                </Link>
+              </div>
+            </div>
+          </div>
+          <span>
+            {props.data.communityCareProvider.practiceName}
+            <br />
+            {props.data.communityCareProvider.firstName} &nbsp;
+            {props.data.communityCareProvider.lastName}
+            <br />
+            {props.data.communityCareProvider.address.street}
+            <br />
+            {props.data.communityCareProvider.address.street2}
+            <br />
+            {props.data.communityCareProvider.address.city}, &nbsp;
+            {props.data.communityCareProvider.address.state} &nbsp;
+            {props.data.communityCareProvider.address.postalCode}
+          </span>
+        </>
+      )}
+      {!props.data.hasCommunityCareProvider && (
+        <>
+          <div className="vads-l-grid-container vads-u-padding--0">
+            <div className="vads-l-row">
+              <div className="vads-l-col--6">
+                <h3 className="vaos-appts__block-label">Preferred providers</h3>
+              </div>
+              <div className="vads-l-col--6 vads-u-text-align--right">
+                <Link
+                  to={newAppointmentFlow.ccPreferences.url}
+                  aria-label="Edit provider preference"
+                >
+                  Edit
+                </Link>{' '}
+              </div>
+            </div>
+          </div>
+          <span>
+            Provider not specified
+            <br />
+            <br />
+            <i>
```

Comment:
I would say that we don't need them to be in italics, because we don't do that elsewhere, but we might want @rtwell to take a look.

Reply:
correct, no need for italics—srry if that styling hung around in an old comp

Reply:
@jbalboni, both text change and removing italics are updated onto the latest commit. Looks like it passed all the tests. Would you approve PR?

Reply:
@jenniemc I still the providers text in the PR, did you push a commit for that?

Reply:
@jbalboni, I am baffled. Under the Commit tab, I see the last commit changed the text to provider and removes the italic. But when I click on File Change tab, the italic tags are removed but text stays 'providers'. Weird....I pushed several more times but no affect.

Reply:
@jbalboni, I found the problem...will fix shortly.